appserver/extras/arquillian-containers/payara-micro/src/test/resources/application.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  JBoss, Home of Professional Open Source
```

Comment:
Why does this file have a JBoss license?

Reply:
This is a left-over from the other connector that shouldn't have been committed for this new one. Removed it.